dts/arm/st/h7/stm32h747.dtsi

```diff
@@ -5,3 +5,16 @@
  */
 
 #include <st/h7/stm32h7.dtsi>
+#include <st/h7/stm32h7_dualcore.dtsi>
+
+/ {
+	cpus {
+		cpu@0 {
+			compatible = "arm,cortex-m7";
+		};
+
+		cpu@1 {
+			compatible = "arm,cortex-m4f";
+		};
+	};
+};
```

Comment:
You'd only need to include `<st/h7/stm32h7_dualcore.dtsi>`

Reply:
Fixed it. Thank you.